packages/medusa/src/api/routes/store/products/get-product.ts

```diff
@@ -76,6 +83,14 @@ export default async (req, res) => {
     relations: defaultStoreProductsRelations,
   })
 
+  let sales_channel_id = validated.sales_channel_id
```

Comment:
todo: if no channel id is provided we should consider defaulting to the default sales channel. Wdyt?

Reply:
Yes! 😄 on it